### ISSUE TITLE
Problems with removeTobiiMatlabSDK branch, no TobiiMex DLL

### ISSUE THREAD

**Issue by iandol** [dcnieho/Titta#6]
Hi, trying to create a TobiiMex() class object I get the following error:

```
>> b=TobiiMex();
Error using TobiiMex.checkMEXFnc (line 32)
Invalid MEX file: "".

Error in TobiiMex (line 68)
            this.mexClassWrapperFnc = this.checkMEXFnc(mexFnc);
```

I can't see a compiled dll file in the folder at the moment...

**Comment by dcnieho** [comment]
Hi Ian,

Yup can't compile the latest changes at home. Will be fixed in a few hours
:)

Closes #6.

Cheers,
Dee

**Comment by iandol** [comment]
Yep, DLL is now in place and all seems to be working...
